src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -55,7 +55,6 @@
 		<PackageReference Include="System.Xml.XmlDocument" Version="4.3.0" />
 		<PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
 		<PackageReference Include="System.Reflection" Version="4.3.0" />
-		<PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
```

Comment:
Also, it would be great to keep the assembly references in both `Castle.Core.Tests.csproj` and `Castle.Core.Tests.WeakNamed.csproj` in sync as far as possible. (Less potential for confusion during future maintenance!)

Reply:
Because `Castle.Core.Tests.csproj` references `Castle.Core.Tests.WeakNamed.csproj` there is no need to list all the packages again in `WeakNamed`.